vagrant/boxes.d/01-builtin.yaml

```diff
@@ -1,23 +1,23 @@
 boxes:
-  centos7-freeipa-server:
-    box: centos7
+  centos9-freeipa-server:
+    box: centos9-stream
     memory: 2048
     ansible:
-      playbook: 'playbooks/freeipa_server.yml'
-      group: 'freeipa_server'
-      server: 'centos7-katello-nightly'
+      playbook: "playbooks/freeipa_server.yml"
+      group: "freeipa_server"
+      server: "centos9-katello-nightly"
```

Comment:
Please don't switch `'` to `"` in this PR. There's no need to and only makes it harder to review.

Reply:
Sorry, that was my editor :/  I'll disable that setting to prevent it from happening again. Thanks for pointing it out.